src/configuration.ts

```diff
@@ -328,6 +328,9 @@ const configuration = {
             .get<string[]>("buildArguments", [])
             .map(substituteVariablesInString);
     },
+    get defaultSwiftVersion(): number | undefined {
```

Comment:
The type of this setting would also have to change to `string` since the valid options are `"6"`, `"5"` and `"Ask Every Run"